File: VideoEditor/ShiftCipher.cpp
#include "ShiftCypher.h"

std::string shift_cipher::encrypt(std::string text, int s)
{
	std::string result = "";

	// traverse text
	for (int i = 0; i < text.length(); i++) {
		// apply transformation to each character
		// Encrypt Uppercase letters

		result += char(text[i] + s);
	}

	// Return the resulting string
	return result;
}

std::string shift_cipher::dectypt(std::string text, int s)
{
	return shift_cipher::encrypt(text, -s);
}

File: shift_cipher.py
def encrypt(text,s):
    result = ""
 
    # traverse text
    for i in range(len(text)):
        char = text[i]
 
        # Encrypt uppercase characters
        result += chr((ord(char) + s))
 
    return result
    
def decrypt(text, s):
    return encrypt(text, -s)